libcudacxx/include/cuda/std/detail/libcxx/include/__config

```diff
@@ -47,17 +47,31 @@
 #  define _LIBCUDACXX_COMPILER_CLANG_CUDA
 #endif
 
-// Some convenience macros to filter nvcc versions
-#if defined(_LIBCUDACXX_COMPILER_NVCC) && _LIBCUDACXX_CUDACC_VER < 1102000
-#define _LIBCUDACXX_COMPILER_NVCC_BELOW_11_2
-#endif // defined(_LIBCUDACXX_COMPILER_NVCC) && _LIBCUDACXX_CUDACC_VER < 1102000
-#if defined(_LIBCUDACXX_COMPILER_NVCC) && _LIBCUDACXX_CUDACC_VER < 1103000
-#define _LIBCUDACXX_COMPILER_NVCC_BELOW_11_3
-#endif // defined(_LIBCUDACXX_COMPILER_NVCC) && _LIBCUDACXX_CUDACC_VER < 1103000
-
-#if defined(_LIBCUDACXX_COMPILER_NVCC) && _LIBCUDACXX_CUDACC_VER < 1108000
-#define _LIBCUDACXX_COMPILER_NVCC_BELOW_11_8
-#endif // defined(_LIBCUDACXX_COMPILER_NVCC) && _LIBCUDACXX_CUDACC_VER < 1108000
+#ifdef __CUDACC__
+#  define _LIBCUDACXX_CUDACC
```

Comment:
This macro now provides the hint that this is a `CUDACC` TU and that CUDA stuff can be assumed to be available.